[2025-01-28T08:42:04.451+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T08:42:04.475+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_sales_data_api manual__2025-01-28T08:34:01.766051+00:00 [queued]>
[2025-01-28T08:42:04.487+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_sales_data_api manual__2025-01-28T08:34:01.766051+00:00 [queued]>
[2025-01-28T08:42:04.488+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 3
[2025-01-28T08:42:04.641+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_sales_data_api> on 2025-01-28 08:34:01.766051+00:00
[2025-01-28T08:42:04.651+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2639) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-28T08:42:04.654+0000] {standard_task_runner.py:72} INFO - Started process 2641 to run task
[2025-01-28T08:42:04.654+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'fetch_sales_data_api', 'manual__2025-01-28T08:34:01.766051+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/raw/dag_load_sales_data_to_db.py', '--cfg-path', '/tmp/tmpthukev2b']
[2025-01-28T08:42:04.655+0000] {standard_task_runner.py:105} INFO - Job 114: Subtask fetch_sales_data_api
[2025-01-28T08:42:04.723+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.fetch_sales_data_api manual__2025-01-28T08:34:01.766051+00:00 [running]> on host 1de8b92bf5f9
[2025-01-28T08:42:04.850+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='fetch_sales_data_api' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T08:34:01.766051+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T08:34:01.766051+00:00'
[2025-01-28T08:42:04.852+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-28T08:42:04.853+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-28T08:42:04.854+0000] {logging_mixin.py:190} INFO - Current task name:fetch_sales_data_api state:running start_date:2025-01-28 08:42:04.476405+00:00
[2025-01-28T08:42:04.854+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2025-01-28T08:42:04.855+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T08:42:06.356+0000] {raw_fetch_sales_api.py:23} INFO - Successfully fetched sales data from the API.
[2025-01-28T08:42:06.379+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-28T08:42:06.392+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T08:42:06.393+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=fetch_sales_data_api, run_id=manual__2025-01-28T08:34:01.766051+00:00, execution_date=20250128T083401, start_date=20250128T084204, end_date=20250128T084206
[2025-01-28T08:42:06.409+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-28T08:42:06.410+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-28T08:42:06.411+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2025-01-28 08:41:54.283955+00:00
[2025-01-28T08:42:06.411+0000] {logging_mixin.py:190} INFO - Task hostname:1de8b92bf5f9 operator:PythonOperator
[2025-01-28T08:42:06.439+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T08:42:06.467+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-28T08:42:06.473+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
